### ISSUE TITLE
Nana: Workflow: 2 weeks

### ISSUE THREAD

**Issue by Nana99999** [HYF-Class20/home#38]
# Week 1 Check-In
## I Need Help With:
 -  Figuring out GitHub
 - I just need to find a structure for myself to follow the lessons ( it is more personal, i guess)
## What went well?
- Creating issues
- Getting used to new operating system for me (Linux)
- Using terminal
## What do you think went less well with you?
- I had a problem with internet on the laptop on the first lesson, so it was hard to follow the class and i lost track, and it is a main reason now for me why I could not complete homeworks in the planned time frame. But I am trying to figure out everything by myself with the videos.
## What have you learned from the first week?
- Basic things on GitHub
  - Creating repository
  - Clone codes
- Using terminal
- Installing programs by terminal 🥹
# Sunday Prep Work
## What have you prepared for next Sunday class?
- I am still in the process of understanding previous assignments

**Comment by bermarte** [comment]
@Nana99999 Don't forget to add week 2.

**Comment by Nana99999** [comment]
@bermarte Thank you! I will do it!🫡
